### ISSUE TITLE
PPA signature absent

### ISSUE THREAD

**Issue by proulxf012** [Stellarium/stellarium#1309]
Hello
Nothing big. Linux Mint 20 user on a Intel i5 recent 14 in. computer.
I am unable to load the PPA in my software sources. I get "signature not available" tag.
If PPA is not available anymore, then you should remove it from the list of available ways to install.
Regards
François Proulx, Longueuil, Québec, Canada

**Comment by alex-w** [comment]
>If PPA is not available anymore, then you should remove it from the list of available ways to install.

All 3 PPA are available and I have installed Stellarium from one of it. If your Mint cannot use PPA anymore then this is not PPA's problem. 

P.S. Please be more accurate and careful - you shared very small technical details and now we should a lot of guessing what is happening on your side.

**Comment by gzotti** [comment]
I guess this is a temporary problem. I was even able to install from PPA into Ubuntu 18.04 on Windows, and can now run Stellarium in WSL with Mesa/LLVM rendering with an Xming server.

**Comment by proulxf012** [comment]
"Sigh..." I was not able to install a few PPAs on the 3rd day after re-installing LM20. I made a mistake that made my previous installation unusable and I re-installed. But I did not do that with "erase disk and install Linux Mint" but used "delete Linux Mint and re-install Linux Mint". Probably a mistake. The first 2 days, I re-installed most of what I needed but the 3rd day, PPAs could no longer be installed. So I finally just sighed and re-installed the whole thing again, this time the right way. Every thing works as advertised again. 
Sorry for disturbing you 
François Proulx, Longueuil, Québec, Canada
